src/dbseeder/sql.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
```

Comment:
I do not like this files name. Maybe something like functions.py or I don't know.